psydac/api/tests/test_2d_multipatch_mapping_poisson.py

```diff
@@ -173,13 +173,9 @@ def test_poisson_2d_2_patches_dirichlet_2():
     D2     = mapping_2(B)
     D3     = mapping_3(C)
 
-    D1D2      = D1.join(D2, name = 'D1D2',
-                bnd_minus = D1.get_boundary(axis=1, ext=1),
-                bnd_plus  = D2.get_boundary(axis=1, ext=-1))
-
-    domain    = D1D2.join(D3, name = 'D1D2D3',
-                bnd_minus = D2.get_boundary(axis=1, ext=1),
-                bnd_plus  = D3.get_boundary(axis=1, ext=-1))
+    connectivity = [((0,1,1),(1,1,-1)), ((1,1,1),(2,1,-1))]
+    domains = [D1, D2, D3]
+    domain = Domain.join(domains, connectivity, 'domain')
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)